conjur/api/__init__.py

```diff
@@ -0,0 +1 @@
+from conjur.api.api import Api
```

Comment:
new line

Reply:
Added lines between comment and imports in the init files